app/assets/stylesheets/components.scss

```diff
@@ -328,6 +328,8 @@
   border-radius: 4px;
   color: $white;
   font-weight: 500;
+  max-width: 75%;
+  overflow: scroll;
```

Comment:
In mobile version, the overflow is likely to be solved. But in desktop there is a unnecessary scroll bar coming.
![image](https://github.com/CircuitVerse/CircuitVerse/assets/57363826/33f75439-43f2-442a-ac30-508fc9329451)

I think, you can switch to `overflow: auto`, that may solve the issue